examples/toy-chat/src/protocol.rs

```diff
@@ -3,7 +3,7 @@ use prost::Message;
 use waku_bindings::{Encoding, WakuContentTopic};
 
 pub static TOY_CHAT_CONTENT_TOPIC: WakuContentTopic =
-    WakuContentTopic::new("toy-chat", 2, "huilong", Encoding::Proto);
+    WakuContentTopic::new("toy-chat", "2", "huilong", Encoding::Proto);
```

Comment:
What about impl `Default` for `WakuContentTopic` instead of the `pub static` ?

Would allow removing the `const` from `new` below.